vllm/worker/multi_step_model_runner.py

```diff
@@ -215,6 +217,46 @@ def prepare_model_input(
         )
         return model_input
 
+    def _async_process_outputs(self, model_input: StatefulModelInput,
+                               output_proc_callback: Callable):
+        # Proceed with pythonization and output_proc in order.
+        # Stop on the first one that fails to pythonize
+        cont = True
+        for model_output in model_input.cached_outputs:
+            if not model_output.pythonized:
```

Comment:
Can remove this line as `maybe_pythonize` checks this internally.

Reply:
this check is needed here because of the "cont", to ensure outputs are processed in order.

Reply:
But you check `model_output.pythonized` again when setting `cont = False` below, so for this case it is redundant?